### PR TITLE
Update Implementation, changed how old runtimes are defined and improved the check for runtime.json

### PR DESCRIPTION
**Pull request by Zemogiter**
### Description
fix https://github.com/PhoenicisOrg/phoenicis/issues/2172
### What works
Shortcuts and running exe files 
### What was not tested
How the script will behave if file in `runtimeJsonPath` is empty.
### Test
- Operating system (and linux kernel version): Ubuntu 19.04 x64 5.3.0-29-generic
- Hardware (GPU/CPU): GTX 1080 ti, i7-7700K

### Ready for review
- [ ] Script tested as a regular phoenicis user and working (if you have a problem -> create as draft and ask for help).
- [ ] `json-align` and `eslint` run according to the [documentation](https://phoenicisorg.github.io/scripts/General/tools/). 
- [x] Codacy and travis checked.

### CODE REVIEW

**Comment by Zemogiter** [comment]
`129:13  error  Parsing error: Assigning to rvalue`

**Comment by madoar** [comment]
@Zemogiter please change your PRs to "Reade for review" when asking for reviews, this makes it easier for us to merge the PR if everything is ok

File: Engines/Wine/Engine/Implementation/script.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ module.default = class WineEngine {
         const localDirectory = this.getLocalDirectory(subCategory, version);
 
         // if not installed
+
```
Comment (madoar) [inline Engines/Wine/Engine/Implementation/script.js]: Please remove this empty line

File: Engines/Wine/Engine/Implementation/script.js [hunk 2]
```diff
@@ -67,6 +68,7 @@ module.default = class WineEngine {
             const wineJson = JSON.parse(this.getAvailableVersions());
 
             wineJson
+
```
Comment (madoar) [inline Engines/Wine/Engine/Implementation/script.js]: Please remove this empty line

File: Engines/Wine/Engine/Implementation/script.js [hunk 3]
```diff
@@ -125,7 +127,8 @@ module.default = class WineEngine {
         let downloadx64 = false;
         let namex86;
         let namex64;
-        if (!fileExists(runtimeJsonPath)) {
+        print ("runtimeJsonPath: " + runtimeJsonPath);
```
Comment (madoar) [inline Engines/Wine/Engine/Implementation/script.js]: Please remove any added test output

File: Engines/Wine/Engine/Implementation/script.js [hunk 4]
```diff
@@ -165,8 +168,10 @@ module.default = class WineEngine {
                 .url("https://phoenicis.playonlinux.com/index.php/runtime?os=linux")
                 .to(runtimeJsonPath)
                 .get();
+            print ("runtimeJsonFile: " + runtimeJsonFile);
```
Comment (madoar) [inline Engines/Wine/Engine/Implementation/script.js]: Please remove any added test output

File: Engines/Wine/Engine/Implementation/script.js [hunk 5]
```diff
@@ -165,8 +168,10 @@ module.default = class WineEngine {
...
 
             runtimeJson = JSON.parse(cat(runtimeJsonFile));
+            print ("runtimeJson: " + runtimeJson);
```
Comment (madoar) [inline Engines/Wine/Engine/Implementation/script.js]: Please remove any added test output

**Comment by Zemogiter** [comment]
@madoar done

**Pull request merged**